odata-parser.pegjs

```diff
@@ -343,8 +343,14 @@ FilterNegateExpression =
 
 GroupedPrimitive =
 	'(' spaces
-		@Primitive|1..,',' spaces|
-	')'
+		bindings:(@Primitive|1..,',' spaces|)
+	spaces ')'
+	{
+		const indices = bindings.map(binding => binding.bind);
+		const bindValues = indices.map(index => binds[index]);
```

Comment:
Let's match what our linter would do if we could run it here:
```suggestion
		const indices = bindings.map((binding) => binding.bind);
		const bindValues = indices.map((index) => binds[index]);
```

Reply:
done